go/proxyd/metrics.go

```diff
@@ -26,9 +27,10 @@ var (
 
 	rpcForwardsTotal = promauto.NewCounterVec(prometheus.CounterOpts{
 		Namespace: MetricsNamespace,
-		Name:      "rpc_backend_requests_total",
+		Name:      "rpc_forwards_total",
 		Help:      "Count of total RPC requests forwarded to each backend.",
 	}, []string{
+		"auth",
```

Comment:
This technically leaks the `auth` value but I suppose the value is super secret anyways

Reply:
Ah, so this won't leak the auth secret - just its alias. You configure auth like this:

```
[authentication]
super-secret-path = "alias"
```

So in this case, `alias` would appear in the monitoring.

I'll update the README with additional context around how to configure the auth path.